recipe/meta.yaml

```
@@ -26,20 +28,31 @@ outputs:
       build:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
-        - {{ compiler("cuda") }}         # [cuda_compiler_version != "None"]
+        - {{ compiler("cuda") }}         # [cuda_compiler_version != "12.0"]
```

Now that CUDA 12 is handled in the migrator, think we can revert this (and re-render) to fix the remaining CI issues

```suggestion
        - {{ compiler("cuda") }}         # [cuda_compiler_version != "None"]
```

---

What about https://github.com/conda-forge/ucx-split-feedstock/pull/117/commits/ce281707ac398454ad683c7da050ae1721dfe5df ? This is how it was done before. Any preferences? We intentionally have not been building non-CUDA because it's redundant, using a CUDA build will work.

---

Although the changes are orthogonal, I think. We want both, right?

---

Ah ok. Yeah the `skip` seems reasonable

Maybe we can drop the selector from this line altogether then?

```suggestion
        - {{ compiler("cuda") }}
```

---

Seems reasonable. Applied this last change now.